app/src/main/res/layout/story_chapter_view.xml

```diff
@@ -76,6 +73,7 @@
             android:layout_width="0dp"
             android:layout_height="wrap_content"
             android:layout_marginStart="8dp"
+            android:layout_marginTop="4dp"
             android:layout_marginEnd="8dp"
             android:fontFamily="sans-serif-medium"
```

Comment:
Add these attributes to `chapter_title`
```
android:ellipsize="end"
android:maxLines="2"
```

Add these attributes to chapter_summary

```
android:ellipsize="end"
android:maxLines="4"
android:minLines="4"
```

Reply:
@rt4914 , all above suggested changes are done. Tick mark icon is also updated as per shown in mock.

Reply:
Thanks.